src/site/content/en/blog/disable-mouse-acceleration/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Disable mouse acceleration to provide a better FPS gaming experience
+subhead: |
+  Web apps can now disable mouse acceleration when capturing pointer events.
+authors:
+  - beaufortfrancois
+date: 2020-10-23
+hero: hero.jpg
+thumbnail: thumbnail.jpg
+alt: Gamer girl is playing FPS online video game on her computer
+description: |
+  Web apps can now disable mouse acceleration when capturing pointer events.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - games
+  - devices
+feedback:
+  - api
+---
+
+An accelerated movement is when the pointer speed increases faster than the
+physical speed of a mouse. In other words, mouse acceleration is what causes the
+pointer to move slowly when moving it a tiny bit, and a farther distance when
+moved fast.
```

Comment:
suggesting: """Accelerated movement is an ergonomic feature when using a mouse or trackpad to move the on screen pointer.  It allows precise movement by moving slowly while also allowing the pointer to cross the entire screen with a quick short motion. Specifically, for the same physical distance that you move the mouse the on screen pointer travels further if the distance was traveled faster."""